openedx/adg/lms/conftest.py

```diff
@@ -21,3 +22,27 @@ def user_with_user_profile(request):
     user = UserFactory()
     UserProfileFactory(user=user)
     return user
+
+
+@pytest.fixture(scope='function', name='get_course')
+def course_fixture():
+    """
+    Creates a function to create a course with the given values in kwargs
+
+    Returns:
+        func: A function that allows user to create a course with the specified kwargs
+    """
+
+    def create_course(**kwargs):
+        """
+        Create a course overview object from the given keyword args if provided
+
+        Arguments:
+            kwargs: Any values to create the course with
+
+        Returns:
+             CourseOverview: A course overview object with the provided values
+        """
+        return CourseOverviewFactory(**kwargs)
+
+    return create_course
```

Comment:
We are not doing anything extra, just returning whatever CourseOverviewFactory, in this case can directly use CourseOverviewFactory, also if the purpose behind implementing it callable is to support the parameters then I'd recommend to go through the [docs of parameterized fixtures.](https://docs.pytest.org/en/stable/fixture.html#fixture-parametrize)